packages/dd-trace/test/datastreams/processor.spec.js

```diff
@@ -203,7 +203,7 @@ describe('DataStreamsProcessor', () => {
     env: 'test',
     version: 'v1',
     service: 'service1',
-    tags: { tag: 'some tag' }
+    tags: { DD_TAG: 'some tag', DD_OTHER_TAG: 'some other tag' }
```

Comment:
`DD_TAG` is not meant to be the tag key right? It's a bit confusing to use it like that in the test.

I would prefer `foo: `foovalue`, bar: `barvalue`

Reply:
fixed